src/bio/bio_monitor.c

```diff
@@ -496,44 +537,6 @@ bio_bs_monitor(struct bio_xs_context *ctxt, uint64_t now)
 	collect_raw_health_data(ctxt);
 }
 
-/* Print the io stat every few seconds, for debug only */
-void
-bio_xs_io_stat(struct bio_xs_context *ctxt, uint64_t now)
-{
-	struct spdk_bdev_io_stat	 stat;
-	struct spdk_bdev		*bdev;
-	struct spdk_io_channel		*channel;
-
-	/* check if IO_STAT_PERIOD environment variable is set */
-	if (io_stat_period == 0)
-		return;
-
-	if (ctxt->bxc_io_stat_age + io_stat_period >= now)
-		return;
-
-	if (ctxt->bxc_desc != NULL) {
-		channel = spdk_bdev_get_io_channel(ctxt->bxc_desc);
-		D_ASSERT(channel != NULL);
-		spdk_bdev_get_io_stat(NULL, channel, &stat);
```

Comment:
This SPDK io statistics is per io channel (xstream), it could be helpful to check if read/write are balanced over all VOS xstreams, maybe we'd add it into telemetry as well in later patch.

If we don't plan to add it to telemetry, the 'bxc_desc' (dedicated for io statistics) could be removed.

Reply:
Got it. I will add a new counter to export the per-io channel stats. Thanks.